.licensei.toml

```
@@ -0,0 +1,10 @@
+approved = [
+  "apache-2.0",
+  "bsd-2-clause",
+  "bsd-3-clause",
+  "isc",
+  "mpl-2.0",
+  "mit",
+]
+
+[header]
```

licensei only seems to support a single header template for go files:
```suggestion
[header]
ignorePaths = ["vendor", "client", ".gen"]
ignoreFiles = ["mock_*.go", "*_gen.go"]
template = """// Copyright (c) OpenTF Foundation.
// SPDX-License-Identifier: MPL-2.0"""

```